Directory.Build.targets

```diff
@@ -9,7 +9,7 @@
   <Import Project="build\import\Workarounds.targets"/>
 
   <ItemGroup>
-    <AdditionalFiles Include="$(MSBuildThisFileDirectory)\src\Common\BannedSymbols.txt" Condition="'$(BannedSymbolsOptOut)' != 'true'" />
```

Comment:
>\ [](start = 57, length = 1)

Just to confirm: this doesn't affect the build itself, right?

Reply:
It just avoids a `\\` in the item spec. It's not a big deal, but potentially results in the string allocations due when getting the canonical path and normalizing separators.